rust/agama-lib/src/product/client.rs

```diff
@@ -135,7 +135,11 @@ impl<'a> ProductClient<'a> {
             .into_iter()
             .map(|(id, version, code)| AddonParams {
                 id,
-                version,
+                version: if version.is_empty() {
```

Comment:
[This function](https://github.com/agama-project/agama/pull/2238#discussion_r2022666760) could be useful in this case. However, it is not included yet.